pkgs/applications/graphics/inkscape/default.nix

```diff
@@ -48,32 +47,36 @@
 let
   python3Env = python3.withPackages
     (ps: with ps; [
-      appdirs
+      # Dependencies listed in CMakeScripts/ConfigCPack.cmake, line starts with set(CPACK_DEBIAN_INKSCAPE-EXTENSIONS_PACKAGE_DEPENDS
```

Comment:
The relevant listing is https://gitlab.com/inkscape/extensions/-/blob/7e13b1b747ef01b240cf8c8581c48a87518370fe/pyproject.toml#L26-39 and in the other pyproject files:

```ShellSession
$ tomlq --slurp 'map(.tool.poetry.dependencies | to_entries | map(.key)) | flatten | unique' $(fd pyproject.toml ~/Projects/inkscape)
[
  "CacheControl",
  "Pillow",
  "PyGObject",
  "appdirs",
  "beautifulsoup4",
  "cssselect",
  "filelock",
  "inkex",
  "lxml",
  "numpy",
  "packaging",
  "pillow",
  "pySerial",
  "pyparsing",
  "python",
  "requests",
  "scour",
  "tinycss2",
  "zstandard"
]
```

The cachecontrol optional dependencies were intentionally next to the cachecontrol.

Reply:
I tried reproducing this by running the command with the [extensions repo](https://gitlab.com/inkscape/extensions) however it only outputs
```
[
  "Pillow",
  "PyGObject",
  "cssselect",
  "lxml",
  "numpy",
  "packaging",
  "pySerial",
  "pyparsing",
  "python",
  "scour",
  "tinycss2"
]
```
The `fd` command only finds one `pyproject.toml` file in the root of the repository. Did I do something incorrectly?

Reply:
Thanks for the tip!